lib/plottr_components/src/components/dashboard/files/FilesHome.js

```diff
@@ -62,20 +63,8 @@ const FilesHomeConnector = (connector) => {
     }
   }
 
-  const FilesHome = ({ errorActions, importActions, isOnWeb }) => {
+  const FilesHome = ({ errorActions, importActions, isOnWeb, projectActions }) => {
     const [view, setView] = useState('recent')
-    const [namingFile, setNamingFile] = useState(null)
```

Comment:
+ 1 million code points for removing this state and simplifying the component.
Great job! 💯